FrontEnd/src/components/AboutPage/MeetTheTeam/MemberInfoCard/MemberInfoCard.tsx

```diff
@@ -1,7 +1,101 @@
 import React, { FunctionComponent } from 'react';
 import { Col, Container, Row } from 'react-bootstrap';
 import styles from './MemberInfoCard.module.scss';
+import Card from 'react-bootstrap/Card';
+import { contactIcons } from '@icons';
+import { Link, FilledImage } from '@basics';
+import { Body1, Body2, Caption } from '@components/basics';
+import cn from 'classnames';
 
-const MemberInfoCard: FunctionComponent = () => <Container></Container>;
+export interface MemberInfoCardProps {
+  name: string;
+  imgSrc: string;
+  title: string;
+  quote: string | { number: number; length: string };
+  website?: string;
+  email?: string;
+  linkedIn?: string;
+}
+
+const MemberInfoCard: FunctionComponent<MemberInfoCardProps> = ({
+  name,
+  imgSrc,
+  title,
+  quote,
+  website,
+  email,
+  linkedIn,
+}) => {
+  const firstName: string = name.split(' ')[0];
+  let team: string = title.toLowerCase();
+  team === 'designer' && (team = 'design');
+
+  const infoCol = (
+    <div className={styles.firstCol}>
+      <div className={styles.information}>
+        <Row>
+          <FilledImage src={imgSrc} alt="Image" className={styles.picture} />
+
+          <Col className={styles.nameInfo}>
+            <div className={styles.name}>
+              <Body1>{name}</Body1>
+            </div>
+
+            <div className={styles.title}>
+              <Caption>{title}</Caption>
+            </div>
+          </Col>
+        </Row>
+        <div className={styles.caption}>
+          {typeof quote === 'string' ? (
+            <Body2>"{quote}"</Body2>
```

Comment:
Is this line on purpose? It looks to me like if someone passes through a string for the quote, all that will show up is the string literal "{quote}" RATHER than the quote itself.